File: example/src/main/java/com/xwray/groupie/example/databinding/item/SmallCardItem.java
package com.xwray.groupie.example.databinding.item;

import android.support.annotation.ColorInt;

public class SmallCardItem extends com.xwray.groupie.example.item.CardItem {

    public SmallCardItem(@ColorInt int colorRes) {
        super(colorRes);
    }

    public SmallCardItem(@ColorInt int colorRes, CharSequence text) {
        super(colorRes, text);
    }

    @Override public int getSpanSize(int spanCount, int position) {
        return spanCount / 3;
    }
}

File: example/src/main/java/com/xwray/groupie/example/item/SmallCardItem.kt
package com.xwray.groupie.example.item

import androidx.annotation.ColorInt

open class SmallCardItem(@ColorInt private val colorInt: Int, text: CharSequence? = "") : CardItem(colorInt, text) {

    override fun getSpanSize(spanCount: Int, position: Int) = spanCount / 3

}
